doc/api/esm.md

```diff
@@ -871,8 +871,7 @@ The resolver can throw the following errors:
 * _Package Import Not Defined_: Package imports do not define the specifier.
 * _Module Not Found_: The package or module requested does not exist.
 
-<details>
-<summary>Resolver algorithm specification</summary>
+### Resolver Algorithm Specification
```

Comment:
Should this be one heading level below `### Resolver Algorithm` above rather than the same level?

```suggestion
#### Resolver Algorithm Specification
```